src/utils/dispute-utils.js

```diff
@@ -4,22 +4,31 @@ import { getTermStartTime } from './court-utils'
 import * as DisputesTypes from '../types/dispute-status-types'
 import { getOutcomeNumber } from './crvoting-utils'
 import { bigNum } from '../lib/math-utils'
+import { getVoidedDisputesByCourt } from '../voided-disputes'
 
 export const FINAL_ROUND_WEIGHT_PRECISION = bigNum(1000)
 export const PCT_BASE = bigNum(10000)
 
 const juryDraftingTerms = 3
 
 export const transformResponseDisputeAttributes = dispute => {
-  return {
+  const voidedDisputes = getVoidedDisputesByCourt()
+  const isDisputeVoided = voidedDisputes.some(
+    voidedDispute => voidedDispute.id === dispute.id
+  )
+
+  const disputeStatus = isDisputeVoided
```

Comment:
could we create a function for this so we don't use those nested `?``:` ?

Reply:
Actually with the comment from @sohkai above this is resolved